docs/RestoreDatabase.md

```diff
@@ -0,0 +1,210 @@
+# Restore database guide
```

Comment:
Following the guidance in [Google Developer Documentation Style Guide ("Heading and title text")](https://developers.google.com/style/headings#heading-and-title-text), how about we use one of the following?:

```suggestion
# Restore a database in a Kubernetes environment
```